setup.py

```diff
@@ -29,12 +29,15 @@ def panther_version() -> str:
         'redis==5.0.1',
         'motor~=3.5.0',
         'bpython~=0.24',
-        'ruff~=0.1.9',
         'python-jose~=3.3.0',
         'websockets~=12.0',
         'cryptography~=42.0.8',
         'watchfiles~=0.21.0',
     ],
+    'dev': [
```

Comment:
Adding the `dev` option was a good choice and you may have to improve it as you develop but let the `ruff` stay in the `full` too

Reply:
hmm
if we were to do that, i think we should make different categories for extra packages, e.g: 'jwt': [ 'python-jose']
so people don't have to install `ruff` in production env

tho i'm not sure what `watchfiles` should be categorised as